src/guides/v2.3/inventory/inventory-api-reference.md

```diff
@@ -66,68 +66,68 @@ The following interfaces and classes have been deprecated. They cannot be mapped
 
 ## {{site.data.var.im}} API
 
-The following services reside under the [`Magento/InventoryApi/Api`](https://github.com/magento/inventory/tree/2.3-develop/app/code/Magento/InventoryApi/Api) namespace, replacing deprecated services and APIs.
+The following services reside under the [`Magento/InventoryApi/Api`](https://github.com/magento/inventory/tree/1.1-develop/app/code/Magento/InventoryApi/Api) namespace, replacing deprecated services and APIs.
```

Comment:
Here and further, there is an open PR that fixes inventory links (#6312). The corresponding changes should be merged there first.

Reply:
The version update seem to be in contradiction with changes in #6312